RELEASE_TEMPLATE.md

```diff
@@ -0,0 +1,253 @@
+## Subject Line
+[Blox] Deploy release - v$$blox_version$$
+
+## Activity Details
+This activity is to: Release Blox version v$$blox_version$$ by merging the dev branch into the master branch on GitHub, and pushing new Blox images up to Docker Hub.
+
+The purpose of this change is to: Release the latest features and bug fixes for Blox to allow consumers to start using the new functionality.
+
+The Blox framework version is: v$$blox_version$$ with a git hash of $$github_hash$$
+
+## Impact Details
+What will happen if this release doesn't happen?  
+Blox consumers will not be able to take advantage of the new features and bug fixes in Blox.
+
+Why is this the correct time/day to complete the release?  
+This corresponds to the completion date of all GitHub issues planned for this release.
+
+Are there any related, prerequisite changes upon which this release hinges?  
+No.
+
+## Worst Case Scenario
+What could happen if this change causes impact?  
+Blox consumers could start running this new version of the Blox framework and encounter random errors leading to unexpected behavior.
+
+Where are the most likely places this change will fail?  
+Errors encountered building the new Blox images or pushing them up to Docker Hub.
+
+## Hostname or Service
+GitHub: https://github.com/blox/blox  
+Docker Hub: https://hub.docker.com/u/bloxoss/
+
+## Timeline / Activity Plan
+Times are relative to the start of the release.
+
+- 00:00 Change the release status to "In Progress".
+- 00:05 Perform the Activity Checklist steps below.
+- 00:25 Perform the Validation Checklist steps below.
+- 01:00 Activity complete. Change the release status to "Complete".
+
+#### Build the bloxoss/cluster-state-service:latest,$$blox_version$$,$$github_hash$$ images and publish to Docker Hub
+```
+$ cd <GitRepoBase>/cluster-state-service/
+$ make release
+$ docker tag bloxoss/cluster-state-service:latest bloxoss/cluster-state-service:$$blox_version$$
+$ docker tag bloxoss/cluster-state-service:latest bloxoss/cluster-state-service:$$github_hash$$
+$ docker push bloxoss/cluster-state-service:latest
+$ docker push bloxoss/cluster-state-service:$$blox_version$$
+$ docker push bloxoss/cluster-state-service:$$github_hash$$
+```
+
+#### Build the bloxoss/daemon-scheduler:latest,$$blox_version$$,$$github_hash$$ images and publish to Docker Hub
+```
+$ cd <GitRepoBase>/daemon-scheduler/
+$ make release
```

Comment:
Yeah, that was me providing a sample without actually looking at the output. We should definitely delete existing bloxoss images and relogin to dockerhub before we push images. I'd like that to be called out explicitly here (with instructions)